astro/src/content/docs/lifecycle/authenticate-users/identity-providers/gaming/discord.mdx

```diff
@@ -22,51 +22,49 @@ Once you have completed this configuration you may enable an OpenID Connect <Inl
 
 ### Register a Discord OAuth2 Application
 
-You will first need to log in to [Discord](https://discord.com/developers/applications/).
-
-Once logged in, navigate to [https://discord.com/developers/applications/](https://discord.com/developers/applications/) and create a new application.
+First, log in to [Discord](https://discord.com/developers/applications/). Then, navigate to [https://discord.com/developers/applications/](https://discord.com/developers/applications/) and create a new application.
 
 <img src="/img/docs/lifecycle/authenticate-users/identity-providers/gaming/discord-openid-connect-client-id-secret.png" alt="Discord Client ID and Secret" width="1200" role="shadowed" />
 
-Once the application has been created, note the `CLIENT ID` and the `CLIENT SECRET`.  These will be used respectively as the <InlineField>Client Id</InlineField> value and the <InlineField>Client secret</InlineField> value in your FusionAuth OpenID Connect Identity Provider configuration.
+Note the `CLIENT ID` and the `CLIENT SECRET` after the application is created. You'll use these to configure the <InlineField>Client Id</InlineField> and <InlineField>Client secret</InlineField> values for your FusionAuth OpenID Connected Identity Provider.
 
-Now navigate to <Breadcrumb>Selected App -> Settings -> OAuth2</Breadcrumb> in the navigation pane on the left side of the screen to configure OAuth2 for your Discord app.
+Finally, configure OAuth2 for your Discord app by opening <Breadcrumb>Selected App -> Settings -> OAuth2</Breadcrumb> in the navigation pane on the left side of the screen.
```

Comment:
These two steps are out of order. You have to first go to the OAuth2 settings page before you can get the `CLIENT_ID` and `CLIENT_SECRET`.

It was like this before, but thought I'd point it out since you're reworking the page.